doc/blobstorage.md

```diff
@@ -0,0 +1,14 @@
+# Cloud Resource Operator - Blob Storage
+
+## Usage
+To seed a Kubernetes/Openshift cluster with an example Blob Storage resource:
+```
+$ make cluster/prepare 
+$ make cluster/seed/blobstorage
+```
+
+## Strategy 
+A config map object is expected to exist for each provider that will be used by the operator. This config map contains provider-specific information about how to deploy a particular resource type, such as blob storage. In the Cloud Resources Operator, this provider-specific configuration is called a strategy. An example of the AWS strategy can be seen [here](../deploy/examples/config_aws_strat.yaml)
```

Comment:
As this block is duplicated, let's move it out to the main README, and have the resource type specific docs just contain specific information